mutalyzer/variant.py

```diff
@@ -0,0 +1,270 @@
+"""
+"""
+
+from __future__ import unicode_literals
+
+from Bio.SeqUtils import seq3
+
+from extractor import extractor
+
+from mutalyzer import models
```

Comment:
I don't like that we are using Spyne models here, they should be very specific to the webservices. If anything, we should try moving these models to the webservice code and remove all use of them elsewhere.

See the `models.py` docstring.